scripts/build.mjs

```diff
@@ -6,6 +8,24 @@ const tsc = process.env.CI || args.includes("--tsc") || publish;
 
 exec("yarn", ["rollup", "-c"])
   .then(() => tsc && exec("yarn", ["tsc", "-b"]))
+  .then(() => {
+    let { LOCAL_BUILD_DIRECTORY } = process.env;
+    if (LOCAL_BUILD_DIRECTORY) {
+      let appDir = path.resolve(LOCAL_BUILD_DIRECTORY);
+      try {
+        fse.readdirSync(path.join(appDir, "node_modules"));
+      } catch {
+        console.error(
+          "Oops! You pointed LOCAL_BUILD_DIRECTORY to a directory that " +
+            "does not have a node_modules/ folder. Please `npm install` in that " +
+            "directory and try again."
+        );
+        process.exit(1);
+      }
+      fse.copySync("build", appDir, { overwrite: true });
+      fse.chmodSync(path.join(appDir, "node_modules/.bin/remix"), 0o755);
+    }
+  })
```

Comment:
I think we lose the `triggerLiveReload` behavior if we just `cp -r` here?  I believe we need to update one of the app files to trigger it, I don't think copying new files into `node_modules` as enough IIRC?

Should we be thinking about the `vite` cache here?  Or should we just be using `--force` in our apps?